CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# Contributing guidelines
+
+If you'd like to contribute to the libmata,  please [fork the repository](https://github.com/VeriFIT/mata/fork),
+create a new feature branch, and finally [create a new pull request](https://github.com/VeriFIT/mata/compare).
+
+In case you run into some unexpected behaviour, error or anything suspicious either contact us directly through mail or
+[create a new issue](https://github.com/VeriFIT/mata/issues/new/choose).
+When creating a new issue, please, try to include everything necessary for us to know (such as the version, operation
+system, etc.) so we can sucessfully replicate the issue.
+
+## Design style
+
+- private attributes of classes/structs should use `snake_case_` (note the trailing `_`).
```

Comment:
Do we actually do that? Maybe put **private** in bold.

Reply:
As per our agreement on one of our previous Mata meetings:
1. we should for all new classes, and
2. we should sometimes rewrite all old classes to adhere to this convention, too.

I shall emphasize the word *private*.